LibertyDSNP/example-client
# Feature/view posts in feed/177294671

Purpose
---------------
The purpose of this PR is to create a new feature that displays posts in the feed.

Solution
---------------
The solution was to use the new post modal and (for the time being) pass the state to the Post component to display new posts in the feed.  I followed the wireframes to style the feed.

Change summary
---------------
* A detailed list of bulleted
* changes that go into detail about
* the specifics of the changes
* to the codebase


Steps to Verify
----------------
1. Read code
2. run the example client locally to see the designs and functionality


Additional details / screenshot
----------------
https://user-images.githubusercontent.com/43625033/124669056-999a7880-de66-11eb-974f-e3ed15e07d9f.mov
